unstructuredstudio/zubhub
# setup pre-commit

pre-commit hooks added:

* https://github.com/psf/black
* https://github.com/PyCQA/isort
* https://github.com/PyCQA/flake8
* https://github.com/pre-commit/pre-commit-hooks
   - trailing-whitespace
   - end-of-file-fixer
   - check-yaml
   - check-shebang-scripts-are-executable
   - check-executables-have-shebangs
   - check-merge-conflict
* https://github.com/pre-commit/mirrors-eslint
* https://github.com/pre-commit/mirrors-prettier
* local
   - zubhub-test
   - media-test
   - react-test

Issue: #996
